pkg/gce-pd-csi-driver/controller.go

```diff
@@ -458,3 +496,52 @@ func diskIsAttachedAndCompatible(volume *compute.Disk, instance *compute.Instanc
 	}
 	return false, nil
 }
+
+func pickTopology(top *csi.TopologyRequirement) (string, error) {
+	reqTop := top.GetRequisite()
+	prefTop := top.GetPreferred()
+
+	// Pick the preferred toplogy in order
+	if len(prefTop) != 0 {
+		if prefTop[0].GetSegments() == nil {
+			return "", fmt.Errorf("preferred toplogies specified specified but no segments")
```

Comment:
"specified" duplicated, and a couple of "toplogy" and "toplogies" in this file

Reply:
done